README.md

```diff
@@ -55,13 +57,33 @@ Any merged PRs with an **Editor Review** or **Reviews Complete** label will be p
 
 ## Building and Validating
 
-If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests. 
-
-1. Download the Mattermost docs repo onto a machine with Python installed.
-2. Install [pipenv](https://docs.pipenv.org/): `pip install pipenv`. 
-3. `cd` into the cloned repository.
-4. Install the required packages: `pipenv install`.
+If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests.
+
+`NOTE: Commands are to be executed on Linux, Mac, and Windows(using Powershell)`
+
+1. Download the Mattermost docs repo onto a machine with Python installed or simple cloned the forked copy of this repo by running,
+```sh
+$ git clone https://github.com/mattermost/docs.git
+```
+2. Install [pipenv](https://docs.pipenv.org/) by following command
+```sh
+$ brew install pipenv   # for Mac users, if you have brew installed)
+```
+```python
+$ pip install pipenv # for all others
+```
+3. Navigate into the cloned repository by following command,
```

Comment:
```suggestion
3. Navigate into the cloned repository:
```

Reply:
Should we also include steps around how to find the built files using Finder/Explorer? Or does this step allow the contributor to select a file and view in a browser?

Reply:
I think this was enough to navigate, as changing directory by `cd` is what is the correct expectations.